reana_client/api/client.py

```diff
@@ -217,7 +217,7 @@ def download_file(self, workflow_id, file_name, access_token):
         :returns: .
         """
         try:
-            logging.getLogger("urllib3").setLevel(logging.CRITICAL)
+            logging.getLogger("requests").setLevel(logging.CRITICAL)
```

Comment:
Hmm, for me this change actually *adds* the warning, not *suppresses* it:

```console
$ reana-client download -w worldpopulation.serial 
[WARNING] Failed to parse headers (url=http://192.168.39.76:32069/api/workflows/worldpopulation.serial/workspace/results%2Fplot.png?access_token=6RhKtqnQk4So2vE091JBtxYkrRaDgZXUMCwwdcm9pjM): [NoBoundaryInMultipartDefect(), MultipartInvariantViolationDefect()], unparsed data: ''
Traceback (most recent call last):
  File "/home/simko/.virtualenvs/reana/lib/python3.7/site-packages/urllib3/connectionpool.py", line 399, in _make_request
    assert_header_parsing(httplib_response.msg)
  File "/home/simko/.virtualenvs/reana/lib/python3.7/site-packages/urllib3/util/response.py", line 66, in assert_header_parsing
    raise HeaderParsingError(defects=defects, unparsed_data=unparsed_data)
urllib3.exceptions.HeaderParsingError: [NoBoundaryInMultipartDefect(), MultipartInvariantViolationDefect()], unparsed data: ''
File results/plot.png downloaded to /home/simko/private/project/reana/src/reana-demo-worldpopulation.
```

With `master` versions` there is no warning.

Perhaps this PR depends on some others, so it should be merged only afterwards?

Reply:
Ah, interesting! It was with master branches everywhere that I was seeing the warning, it must be my python version then.

Reply:
Please double check then whether it works on your end from scratch. Anyway, the rest of the changes in the PR are good, so after you double check on your end, even if you don't need to change the urllib thing, it would be still good to rebase and merge the rest.

Reply:
Indeed this adds the warning back in python 3.6.6 and 3.7.1. I will test once more from my desktop where I saw the issue and inform if I see any differences there. Rephrased the PR to mention minor cli improvements.